starter-workflows/code-scanning/frogbot-scan-and-fix.yml

```diff
@@ -7,14 +7,13 @@
 # Uses JFrog Xray to scan the project.
 # Read more about Frogbot here - https://github.com/jfrog/frogbot#frogbot
 
+# Frogbot uses a frogbot-config.yml file to run. The following article will guide you through the process of creating this file:
+# https://github.com/jfrog/frogbot/blob/master/docs/frogbot-config.md
+
```

Comment:
Is it really needed for the starter workflow? It should be minimal and without customization.
Same question for scan-pr.